src/micromamba/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@ target_link_libraries(micromamba PRIVATE mamba-static)
 message("Using static dependencies for micromamba: ${STATIC_DEPENDENCIES}")
 
 if (STATIC_DEPENDENCIES AND UNIX)
+    add_definitions(-DUMAMBA_STATIC)
```

Comment:
this is the wrong place to add it, isn't it? we're not building the library here? Am I missing something?

Reply:
it's totally the wrong place, sorry for that